### PR TITLE
Sgt toggle component added

### PR DESCRIPTION
**Pull request by sujit-h-singh**
Signed-off-by: Sujit Singh <sujit.h.singh@seagate.com>

# Problem Statement
- toggle component created to match the UX design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/components/manage/s3/LrS3Access.vue [hunk 1]
```diff
@@ -113,13 +117,17 @@ export default class LrS3Access extends Vue {
...
   generatedAccessKeyDialog = false;
   deleteModal = create<SgtDialogModel>(SgtDialog);
+  status = "active";
 
   mounted() {
     this.getAccessKeys();
   }
   getAccessKeys() {
     Api.getData("s3/access_keys", { isDummy: true }).then((resp: any) => {
       this.accessList = resp["access_keys"];
```
Comment (NareshGopal) [inline src/components/manage/s3/LrS3Access.vue:127]: This line can be removed.

File: src/components/manage/s3/LrS3Access.vue [hunk 2]
```diff
@@ -113,13 +117,17 @@ export default class LrS3Access extends Vue {
...
+      this.accessList = resp["access_keys"].map((ele: any) => {
+        return { ...ele, statusFlag: ele.status == "active" ? true : false };
```
Comment (NareshGopal) [inline src/components/manage/s3/LrS3Access.vue:129]: Would suggest using strict equality check '===' and you could also remove the ternary operator for returning boolean values as the equality check operator itself returns a boolean value.

**Review by NareshGopal — APPROVED** [review]

File: src/components/manage/s3/LrS3Access.vue (hunk 1 above)
Comment (sujit-h-singh) [reply src/components/manage/s3/LrS3Access.vue]: done

File: src/components/manage/s3/LrS3Access.vue (hunk 2 above)
Comment (sujit-h-singh) [reply src/components/manage/s3/LrS3Access.vue]: done

**Pull request merged**